synapse/handlers/sso.py

```diff
@@ -29,6 +30,7 @@
 from urllib.parse import urlencode
 
 import attr
+import requests
```

Comment:
We use Twisted as a web framework, `requests` is not really used directly in our code outside of small helper scripts.

You can find an example to how to use Twisted through an existing helper [here](https://github.com/matrix-org/synapse/blob/develop/synapse/handlers/oidc.py#L570).

Reply:
Thanks for the example, I am also using the instantiated client now.